### DISCUSSION TITLE
怎么查询一段时间内每天的第一条轨迹

### DISCUSSION THREAD

**Discussion by yigexiaopianzi** [taosdata/TDengine#4594] [General]
请问一下TDengine中有 FROM_UNIXTIME()这样的语法吗 ， 轨迹表里面有到毫秒的时间戳的列，我要怎么查出指定时间段内的每天的第一条轨迹尼？

**Comment by sangshuduo** [comment]
用 first() 和 interval(1d) 可以吗？

**Reply by yigexiaopianzi** [reply]
select first(*) from stb_feima_track s  where  location_time >= 1598803200000 and 1601481599999>= location_time  interval(1d) group by employee_id   你好  我这么写报错了 interval not allowed in group by normal column   
![image](https://user-images.githubusercontent.com/36226237/102455375-7949ce80-407a-11eb-8ce7-aee1db2a5c6e.png)
  我要拿到9月份每个员工每天的最后一条轨迹    这个sql有问题  请问一下  这个sql要怎么改

**Reply by yigexiaopianzi** [reply]
这样子不行，试过了

**Reply by sangshuduo** [reply]
一个 employee 一个表就可以解决了

**Reply by yigexiaopianzi** [reply]
搞定了  谢谢
